questions/includes/fedora/coding.yml

```diff
@@ -2,30 +2,38 @@ tree:
   children:
     - title: Python
       subtitle: a dynamic language for rapid development
-      segue1: So, you enjoy traceback-driven development?
+      segue1: So, you enjoy traceback-driven development?  What could go wrong?
       segue2: Are you interested in hacking on...
       children: coding/python.yml
 
     - title: C
       subtitle: close to the metal
-      href: https://lkml.org
+      segue1: So, you think OOP is for hipsters?  Let me tell you, back in my day...
+      segue2: How about working on...
+      children: coding/c.yml
 
     - title: Haskell
       subtitle: a nondynamic language for rapid development
-      segue1: Yay, types!
+      segue1: So, programming without side-effects is what you're all about?  Sounds useful!
```

Comment:
Well, https://www.youtube.com/watch?v=iSmkqocn0oQ :P

If you wanted, you could change it to something "punchy" like "If it typechecks, it OBVIOUSLY works. Wink-wink." But even though it's a joke, it'll probably get misconstrued.

Or you could have some fun with vocabulary that we often throw around in Haskell land:

Something like "Where the monads are just free-monad monad monad-algebras -- monad-algebras of the free-monad monad." (c/o https://twitter.com/kmett/status/384766778062016512)

Or something involving Haskell being lazily evaluated, perhaps. "So you like being lazy, huh?"

There's quite a few routes we can go :P